geode-gfsh/src/main/java/org/apache/geode/management/internal/cli/commands/CreateRegionCommand.java

```diff
@@ -168,6 +169,8 @@ public ResultModel createRegion(
           help = CliStrings.CREATE_REGION__LOCALMAXMEMORY__HELP) Integer prLocalMaxMemory,
       @CliOption(key = CliStrings.CREATE_REGION__OFF_HEAP, specifiedDefaultValue = "true",
           help = CliStrings.CREATE_REGION__OFF_HEAP__HELP) Boolean offHeap,
+      @CliOption(key = CliStrings.CREATE_REGION__PARTITION_LISTENER,
```

Comment:
looks like the region can have multiple partition listeners, so `ClassName[]` can be used here, you will need to add `optionContext = "splittingRegex=,(?![^{]*\\})",` to allow splitting the input only with "," outside of json string.